Broken v6.1.0 release

Hello,
I'm facing the issue:

```
17:58:03 Cannot find module 'traverse'
17:58:03 Require stack:
17:58:03 - /mnt/project/node_modules/aws-sdk-mock/dist/index.js
```

The `traverse` module has been replaced at:

- https://github.com/dwyl/aws-sdk-mock/commit/b6fdd98624ba906cb71d72ff8f1401147af98292

But the npm released version is not compiled correctly:

https://www.npmjs.com/package/aws-sdk-mock?activeTab=code

<img width="477" alt="image" src="https://github.com/user-attachments/assets/41e3ec1d-a9d0-4bc7-9bf6-bcabfc66ea44">

If I check the compiled version locally, the correct build looks like:

<img width="493" alt="image" src="https://github.com/user-attachments/assets/1a8ee1ec-439e-4644-8c0a-fd363b08c4a0">

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "aws-sdk-mock",
-  "version": "6.1.0",
+  "version": "6.1.1",
   "description": "Functions to mock the JavaScript aws-sdk",
   "main": "dist/index.js",
   "module": "dist/index.mjs",

--- a/package.json
+++ b/package.json
@@ -15,6 +15,7 @@
     "nocov": "ts-mocha test/**/*.spec.ts",
     "test": "nyc ts-mocha test/**/*.spec.ts && tsd",
     "coverage": "nyc --report html ts-mocha test/**/*.spec.ts && open coverage/index.html",
+    "prepublishOnly": "npm run build",
     "build": "tsup"
   },
   "repository": {

